orstab.tex

```diff
@@ -18,15 +18,17 @@
 	Dress rehearsal -- Just like it will be April for the actual commissioning.
 	\\ \hline
 
-2021 &  NCSA &  \textbf{Operations rehearsal for data release processing (regular data).}
 	\\ \hline
 
-Feb 2022 &  NCSA/Base &  \textbf{Operations rehearsal}
-Rehearsals for real operations which start Oct 2022
+Nov 2021 &  NCSA/Base &  \textbf{Operations rehearsal}
+Rehearsals for real operations which start  Nov 2021
 	\\ \hline
-Sept 2022 &  NCSA/Base &  \textbf{Operations rehearsal}
-Full Dress rehearsal for real operations which start Oct 2022
+Jan 2022 &  NCSA/Base &  \textbf{Operations rehearsal data release processing}
+Full Dress rehearsal for real data release
 	\\ \hline
 
+Aug 2022 &  NCSA &  \textbf{Operations rehearsal for full data release processing (if needed).}
```

Comment:
Is this supposed to align with the Sept 2022 date for LDM-503-17? At the moment, it seems to be a month earlier.